packages/gatsby-source-lever/src/gatsby-node.js

```diff
@@ -3,6 +3,50 @@ const normalize = require(`./normalize`)
 
 const typePrefix = `lever__`
 
+exports.createSchemaCustomization = async ({ actions }) => {
+  const typeDefs = `
+    type lever implements Node {
+      additional: String
+      additionalPlain: String
+      applyUrl: String
+      createdAt: Date
+      description: String
+      descriptionPlain: String
+      id: ID!
```

Comment:
ID is added automatically by `Gatsby` for every type implementing `Node` interface.

```suggestion
```